### ISSUE TITLE
Fix March on Washington Exhibit Page Lists

### ISSUE THREAD

**Issue by Muraszko** [afred/openvault#1198]
Once again, the re-ingest broke these static links, I'll provide UOI_IDs for what to populate the tabs with.

**Done When**
Provided list of UOI_IDs populate the tabs on the exhibit page for March on Washington.

**Comment by Muraszko** [comment]
Please keep the tabs and the text for the links the same:

ERN's Coverage
```
P1: 981fb0f23384faa001638cadeab4a06a8286126d
P2: 6e92a7d9a1b580715e4c0e94b2df4deb4c460464
P3: 44bf881e597213dc09db92197cce20b78344612e
P4: ea85b8f6b92a97538889b8f1cb8d6cc10228781f
P5: 09b4076501f6de215e2405e224be955787bc109d
P6: 83044b7eaca09bc59be09303a020848028592217
P7: af1856d1471a0f632fa20159245a803e8d68f85e
P8: 902a099afb869e2f8bf6e68137f02d04b5777724

**Comment by afred** [comment]
fixed by #1199. @Muraszko take a look on [staging](http://openvault-staging.wgbh.org/collections/march-on-washington). If it looks good, please close this ticket and let me know. I'll deploy to production.

**Comment by Muraszko** [comment]
@afred 
Looks good, feel free to deploy.
